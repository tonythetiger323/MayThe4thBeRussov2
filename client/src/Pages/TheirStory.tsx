import React from 'react'
import MeganCard from '../Components/MeganCard'
import RandyCard from '../Components/RandyCard'
import { Image, Transformation } from 'cloudinary-react'
import Grid from '@material-ui/core/Grid'
import Card from '@material-ui/core/Card'
import CardContent from '@material-ui/core/CardContent'
import Typography from '@material-ui/core/Typography'
import Responsive from 'react-responsive'

const Mobile = (props: any) => <Responsive {...props} maxWidth={897} />
const Default = (props: any) => <Responsive {...props} minWidth={898}/>

const TheirStory = () => {
  return(
    <div>
      <Grid container direction='column' alignItems='center'>
        <br/>
        <MeganCard/>
        <br/>
        <RandyCard/>
        <br/>
        <Default>
          <Grid container alignItems='center' direction='column'>
            <Grid item>
              <Image cloudName='tonythetiger323' publicId='MayThe4thBeRusso/Their Story Page/IMG_0006.jpg' >
                <Transformation height='500' crop='scale' />
              </Image>
            </Grid>
            <Grid item>
              <Card>
                <CardContent>
                  <Typography variant='body1' align='center' style={{ fontFamily: 'Oswald' }}>If you didn't already guess it, Megan said yes. Megan and Randy will be getting married on May 4th, 2019 in Hawaii. They are so excited to tie the knot and start the next chapter of their life together. 
                  </Typography>
                  <Typography variant='body1' align='center' style={{ fontFamily: 'Oswald' }}>
                  Thank you for visiting our site.
                  </Typography>
                </CardContent>
              </Card>
            </Grid>
          </Grid>
        </Default>

        <Mobile>
          <Image cloudName='tonythetiger323' publicId='MayThe4thBeRusso/Their Story Page/IMG_0006.jpg' >
            <Transformation height='460' crop='scale' />
          </Image>
          <Card>
            <CardContent>
              <Typography variant='body1' align='center' style={{ fontFamily: 'Oswald' }}>If you didn't already guess it, Megan said yes. Megan and Randy will be getting married on May 4th, 2019 in Hawaii. They are so excited to tie the knot and start the next chapter of their life together. 
              </Typography>
              <Typography variant='body1' align='center' style={{ fontFamily: 'Oswald' }}>
              Thank you for visiting our site.
              </Typography>
            </CardContent>
          </Card>
        </Mobile>
      </Grid>
    </div>
  )
}

export default TheirStory