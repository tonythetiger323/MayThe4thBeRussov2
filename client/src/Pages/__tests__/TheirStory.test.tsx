import * as React from 'react'
import { shallow } from 'enzyme'
import TheirStory from '../TheirStory'

describe('TheirStory Page', () => {
  const theirStory = shallow(<TheirStory />)

  it('should render correctly', () => {
    expect(theirStory).toMatchSnapshot()
  })
})
