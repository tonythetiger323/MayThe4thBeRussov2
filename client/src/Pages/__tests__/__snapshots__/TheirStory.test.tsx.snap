// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheirStory Page should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TheirStory />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <WithStyles(Typography)
          align="center"
          variant="h2"
        >
          Their Story
        </WithStyles(Typography)>,
        <WithStyles(Grid)
          container={true}
          direction="row"
        >
          <WithStyles(Grid)
            item={true}
            xs={3}
          >
            <CloudinaryContext>
              <Image
                publicId="MayThe4thBeRusso/Their Story Page/IMG_0006.jpg"
              >
                <Transformation
                  crop="scale"
                  quality="100"
                  width="400"
                />
              </Image>
            </CloudinaryContext>
          </WithStyles(Grid)>
          <WithStyles(Grid)
            item={true}
            xs={1}
          />
          <WithStyles(Grid)
            item={true}
            xs={8}
          >
            <WithStyles(Paper)>
              Ex aliquip veniam sit id dolore exercitation nisi. Cupidatat sit eu labore quis. Laborum duis sit ea id duis. Nisi officia esse deserunt labore fugiat labore do deserunt. Lorem ex laboris magna deserunt eiusmod ullamco sint eiusmod eiusmod. Consectetur cupidatat do nisi mollit elit eu consectetur do tempor. Irure labore do eu excepteur ullamco veniam tempor esse sit sint aute cupidatat ipsum. Anim et mollit adipisicing quis. Fugiat nostrud veniam nisi amet excepteur sit aute enim. Aute enim pariatur consequat ea. Tempor quis esse in velit consectetur amet aliquip. Voluptate ea anim fugiat cupidatat ipsum laboris fugiat laborum velit cupidatat qui qui do. Duis proident aliquip do commodo velit commodo ut duis sunt deserunt. Tempor aliquip duis laborum consequat et ullamco dolor dolore qui incididunt. Ullamco elit commodo quis est. Voluptate laborum do ut nostrud irure ex nostrud dolore.
            </WithStyles(Paper)>
          </WithStyles(Grid)>
        </WithStyles(Grid)>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "align": "center",
          "children": "Their Story",
          "variant": "h2",
        },
        "ref": null,
        "rendered": "Their Story",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <WithStyles(Grid)
              item={true}
              xs={3}
            >
              <CloudinaryContext>
                <Image
                  publicId="MayThe4thBeRusso/Their Story Page/IMG_0006.jpg"
                >
                  <Transformation
                    crop="scale"
                    quality="100"
                    width="400"
                  />
                </Image>
              </CloudinaryContext>
            </WithStyles(Grid)>,
            <WithStyles(Grid)
              item={true}
              xs={1}
            />,
            <WithStyles(Grid)
              item={true}
              xs={8}
            >
              <WithStyles(Paper)>
                Ex aliquip veniam sit id dolore exercitation nisi. Cupidatat sit eu labore quis. Laborum duis sit ea id duis. Nisi officia esse deserunt labore fugiat labore do deserunt. Lorem ex laboris magna deserunt eiusmod ullamco sint eiusmod eiusmod. Consectetur cupidatat do nisi mollit elit eu consectetur do tempor. Irure labore do eu excepteur ullamco veniam tempor esse sit sint aute cupidatat ipsum. Anim et mollit adipisicing quis. Fugiat nostrud veniam nisi amet excepteur sit aute enim. Aute enim pariatur consequat ea. Tempor quis esse in velit consectetur amet aliquip. Voluptate ea anim fugiat cupidatat ipsum laboris fugiat laborum velit cupidatat qui qui do. Duis proident aliquip do commodo velit commodo ut duis sunt deserunt. Tempor aliquip duis laborum consequat et ullamco dolor dolore qui incididunt. Ullamco elit commodo quis est. Voluptate laborum do ut nostrud irure ex nostrud dolore.
              </WithStyles(Paper)>
            </WithStyles(Grid)>,
          ],
          "container": true,
          "direction": "row",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <CloudinaryContext>
                <Image
                  publicId="MayThe4thBeRusso/Their Story Page/IMG_0006.jpg"
                >
                  <Transformation
                    crop="scale"
                    quality="100"
                    width="400"
                  />
                </Image>
              </CloudinaryContext>,
              "item": true,
              "xs": 3,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <Image
                  publicId="MayThe4thBeRusso/Their Story Page/IMG_0006.jpg"
                >
                  <Transformation
                    crop="scale"
                    quality="100"
                    width="400"
                  />
                </Image>,
                "cloudName": undefined,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <Transformation
                    crop="scale"
                    quality="100"
                    width="400"
                  />,
                  "publicId": "MayThe4thBeRusso/Their Story Page/IMG_0006.jpg",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "crop": "scale",
                    "quality": "100",
                    "width": "400",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                "type": [Function],
              },
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "item": true,
              "xs": 1,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <WithStyles(Paper)>
                Ex aliquip veniam sit id dolore exercitation nisi. Cupidatat sit eu labore quis. Laborum duis sit ea id duis. Nisi officia esse deserunt labore fugiat labore do deserunt. Lorem ex laboris magna deserunt eiusmod ullamco sint eiusmod eiusmod. Consectetur cupidatat do nisi mollit elit eu consectetur do tempor. Irure labore do eu excepteur ullamco veniam tempor esse sit sint aute cupidatat ipsum. Anim et mollit adipisicing quis. Fugiat nostrud veniam nisi amet excepteur sit aute enim. Aute enim pariatur consequat ea. Tempor quis esse in velit consectetur amet aliquip. Voluptate ea anim fugiat cupidatat ipsum laboris fugiat laborum velit cupidatat qui qui do. Duis proident aliquip do commodo velit commodo ut duis sunt deserunt. Tempor aliquip duis laborum consequat et ullamco dolor dolore qui incididunt. Ullamco elit commodo quis est. Voluptate laborum do ut nostrud irure ex nostrud dolore.
              </WithStyles(Paper)>,
              "item": true,
              "xs": 8,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Ex aliquip veniam sit id dolore exercitation nisi. Cupidatat sit eu labore quis. Laborum duis sit ea id duis. Nisi officia esse deserunt labore fugiat labore do deserunt. Lorem ex laboris magna deserunt eiusmod ullamco sint eiusmod eiusmod. Consectetur cupidatat do nisi mollit elit eu consectetur do tempor. Irure labore do eu excepteur ullamco veniam tempor esse sit sint aute cupidatat ipsum. Anim et mollit adipisicing quis. Fugiat nostrud veniam nisi amet excepteur sit aute enim. Aute enim pariatur consequat ea. Tempor quis esse in velit consectetur amet aliquip. Voluptate ea anim fugiat cupidatat ipsum laboris fugiat laborum velit cupidatat qui qui do. Duis proident aliquip do commodo velit commodo ut duis sunt deserunt. Tempor aliquip duis laborum consequat et ullamco dolor dolore qui incididunt. Ullamco elit commodo quis est. Voluptate laborum do ut nostrud irure ex nostrud dolore.",
              },
              "ref": null,
              "rendered": "Ex aliquip veniam sit id dolore exercitation nisi. Cupidatat sit eu labore quis. Laborum duis sit ea id duis. Nisi officia esse deserunt labore fugiat labore do deserunt. Lorem ex laboris magna deserunt eiusmod ullamco sint eiusmod eiusmod. Consectetur cupidatat do nisi mollit elit eu consectetur do tempor. Irure labore do eu excepteur ullamco veniam tempor esse sit sint aute cupidatat ipsum. Anim et mollit adipisicing quis. Fugiat nostrud veniam nisi amet excepteur sit aute enim. Aute enim pariatur consequat ea. Tempor quis esse in velit consectetur amet aliquip. Voluptate ea anim fugiat cupidatat ipsum laboris fugiat laborum velit cupidatat qui qui do. Duis proident aliquip do commodo velit commodo ut duis sunt deserunt. Tempor aliquip duis laborum consequat et ullamco dolor dolore qui incididunt. Ullamco elit commodo quis est. Voluptate laborum do ut nostrud irure ex nostrud dolore.",
              "type": [Function],
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <WithStyles(Typography)
            align="center"
            variant="h2"
          >
            Their Story
          </WithStyles(Typography)>,
          <WithStyles(Grid)
            container={true}
            direction="row"
          >
            <WithStyles(Grid)
              item={true}
              xs={3}
            >
              <CloudinaryContext>
                <Image
                  publicId="MayThe4thBeRusso/Their Story Page/IMG_0006.jpg"
                >
                  <Transformation
                    crop="scale"
                    quality="100"
                    width="400"
                  />
                </Image>
              </CloudinaryContext>
            </WithStyles(Grid)>
            <WithStyles(Grid)
              item={true}
              xs={1}
            />
            <WithStyles(Grid)
              item={true}
              xs={8}
            >
              <WithStyles(Paper)>
                Ex aliquip veniam sit id dolore exercitation nisi. Cupidatat sit eu labore quis. Laborum duis sit ea id duis. Nisi officia esse deserunt labore fugiat labore do deserunt. Lorem ex laboris magna deserunt eiusmod ullamco sint eiusmod eiusmod. Consectetur cupidatat do nisi mollit elit eu consectetur do tempor. Irure labore do eu excepteur ullamco veniam tempor esse sit sint aute cupidatat ipsum. Anim et mollit adipisicing quis. Fugiat nostrud veniam nisi amet excepteur sit aute enim. Aute enim pariatur consequat ea. Tempor quis esse in velit consectetur amet aliquip. Voluptate ea anim fugiat cupidatat ipsum laboris fugiat laborum velit cupidatat qui qui do. Duis proident aliquip do commodo velit commodo ut duis sunt deserunt. Tempor aliquip duis laborum consequat et ullamco dolor dolore qui incididunt. Ullamco elit commodo quis est. Voluptate laborum do ut nostrud irure ex nostrud dolore.
              </WithStyles(Paper)>
            </WithStyles(Grid)>
          </WithStyles(Grid)>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "align": "center",
            "children": "Their Story",
            "variant": "h2",
          },
          "ref": null,
          "rendered": "Their Story",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <WithStyles(Grid)
                item={true}
                xs={3}
              >
                <CloudinaryContext>
                  <Image
                    publicId="MayThe4thBeRusso/Their Story Page/IMG_0006.jpg"
                  >
                    <Transformation
                      crop="scale"
                      quality="100"
                      width="400"
                    />
                  </Image>
                </CloudinaryContext>
              </WithStyles(Grid)>,
              <WithStyles(Grid)
                item={true}
                xs={1}
              />,
              <WithStyles(Grid)
                item={true}
                xs={8}
              >
                <WithStyles(Paper)>
                  Ex aliquip veniam sit id dolore exercitation nisi. Cupidatat sit eu labore quis. Laborum duis sit ea id duis. Nisi officia esse deserunt labore fugiat labore do deserunt. Lorem ex laboris magna deserunt eiusmod ullamco sint eiusmod eiusmod. Consectetur cupidatat do nisi mollit elit eu consectetur do tempor. Irure labore do eu excepteur ullamco veniam tempor esse sit sint aute cupidatat ipsum. Anim et mollit adipisicing quis. Fugiat nostrud veniam nisi amet excepteur sit aute enim. Aute enim pariatur consequat ea. Tempor quis esse in velit consectetur amet aliquip. Voluptate ea anim fugiat cupidatat ipsum laboris fugiat laborum velit cupidatat qui qui do. Duis proident aliquip do commodo velit commodo ut duis sunt deserunt. Tempor aliquip duis laborum consequat et ullamco dolor dolore qui incididunt. Ullamco elit commodo quis est. Voluptate laborum do ut nostrud irure ex nostrud dolore.
                </WithStyles(Paper)>
              </WithStyles(Grid)>,
            ],
            "container": true,
            "direction": "row",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <CloudinaryContext>
                  <Image
                    publicId="MayThe4thBeRusso/Their Story Page/IMG_0006.jpg"
                  >
                    <Transformation
                      crop="scale"
                      quality="100"
                      width="400"
                    />
                  </Image>
                </CloudinaryContext>,
                "item": true,
                "xs": 3,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <Image
                    publicId="MayThe4thBeRusso/Their Story Page/IMG_0006.jpg"
                  >
                    <Transformation
                      crop="scale"
                      quality="100"
                      width="400"
                    />
                  </Image>,
                  "cloudName": undefined,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <Transformation
                      crop="scale"
                      quality="100"
                      width="400"
                    />,
                    "publicId": "MayThe4thBeRusso/Their Story Page/IMG_0006.jpg",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "crop": "scale",
                      "quality": "100",
                      "width": "400",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  "type": [Function],
                },
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "item": true,
                "xs": 1,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <WithStyles(Paper)>
                  Ex aliquip veniam sit id dolore exercitation nisi. Cupidatat sit eu labore quis. Laborum duis sit ea id duis. Nisi officia esse deserunt labore fugiat labore do deserunt. Lorem ex laboris magna deserunt eiusmod ullamco sint eiusmod eiusmod. Consectetur cupidatat do nisi mollit elit eu consectetur do tempor. Irure labore do eu excepteur ullamco veniam tempor esse sit sint aute cupidatat ipsum. Anim et mollit adipisicing quis. Fugiat nostrud veniam nisi amet excepteur sit aute enim. Aute enim pariatur consequat ea. Tempor quis esse in velit consectetur amet aliquip. Voluptate ea anim fugiat cupidatat ipsum laboris fugiat laborum velit cupidatat qui qui do. Duis proident aliquip do commodo velit commodo ut duis sunt deserunt. Tempor aliquip duis laborum consequat et ullamco dolor dolore qui incididunt. Ullamco elit commodo quis est. Voluptate laborum do ut nostrud irure ex nostrud dolore.
                </WithStyles(Paper)>,
                "item": true,
                "xs": 8,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Ex aliquip veniam sit id dolore exercitation nisi. Cupidatat sit eu labore quis. Laborum duis sit ea id duis. Nisi officia esse deserunt labore fugiat labore do deserunt. Lorem ex laboris magna deserunt eiusmod ullamco sint eiusmod eiusmod. Consectetur cupidatat do nisi mollit elit eu consectetur do tempor. Irure labore do eu excepteur ullamco veniam tempor esse sit sint aute cupidatat ipsum. Anim et mollit adipisicing quis. Fugiat nostrud veniam nisi amet excepteur sit aute enim. Aute enim pariatur consequat ea. Tempor quis esse in velit consectetur amet aliquip. Voluptate ea anim fugiat cupidatat ipsum laboris fugiat laborum velit cupidatat qui qui do. Duis proident aliquip do commodo velit commodo ut duis sunt deserunt. Tempor aliquip duis laborum consequat et ullamco dolor dolore qui incididunt. Ullamco elit commodo quis est. Voluptate laborum do ut nostrud irure ex nostrud dolore.",
                },
                "ref": null,
                "rendered": "Ex aliquip veniam sit id dolore exercitation nisi. Cupidatat sit eu labore quis. Laborum duis sit ea id duis. Nisi officia esse deserunt labore fugiat labore do deserunt. Lorem ex laboris magna deserunt eiusmod ullamco sint eiusmod eiusmod. Consectetur cupidatat do nisi mollit elit eu consectetur do tempor. Irure labore do eu excepteur ullamco veniam tempor esse sit sint aute cupidatat ipsum. Anim et mollit adipisicing quis. Fugiat nostrud veniam nisi amet excepteur sit aute enim. Aute enim pariatur consequat ea. Tempor quis esse in velit consectetur amet aliquip. Voluptate ea anim fugiat cupidatat ipsum laboris fugiat laborum velit cupidatat qui qui do. Duis proident aliquip do commodo velit commodo ut duis sunt deserunt. Tempor aliquip duis laborum consequat et ullamco dolor dolore qui incididunt. Ullamco elit commodo quis est. Voluptate laborum do ut nostrud irure ex nostrud dolore.",
                "type": [Function],
              },
              "type": [Function],
            },
          ],
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
